.github/workflows/buid_and_release.yml

```diff
@@ -39,3 +39,5 @@ jobs:
           output/release/medgen.obo
           output/release/medgen-disease-extract.obo
           output/release/medgen-xrefs.robot.template.tsv
+          output/release/hpo-umls.sssom.tsv
+          output/release/hpo-mesh.sssom.tsv
```

Comment:
## Question: Release only what we're using?
- [ ] @matentzn 

Just double checking; I know we're using these files:
- `output/release/medgen-xrefs.robot.template.tsv`
- `output/release/hpo-umls.sssom.tsv`
- `output/release/hpo-mesh.sssom.tsv`

But what about these? If we're not using these, should they still be in the release?
- `output/release/medgen.obo`
- `output/release/medgen-disease-extract.obo`

Reply:
I'll keep em in.